lib/cdo/lesson_import_helper.rb

```diff
@@ -102,51 +96,45 @@ def self.create_activity_sections(activity_markdown, lesson_activity_id, levels)
 
     sections = []
     name = ''
-    position = 1
     sorted_matches.each do |match|
-      activity_section = nil
-      if match[:type] == 'skippable'
-        next
-      elsif match[:type] == 'tip'
+      if match[:type] == 'tip'
         activity_section = ActivitySection.new
         key = match[:match][3]&.strip || "#{match[:match][1]}-0"
         activity_section.tips = [create_tip(key, match[:match][1] || "tip", match[:match][4])]
         activity_section.key ||= SecureRandom.uuid
+        activity_sections = [activity_section]
         match[:activity_section_key] = activity_section.key
       elsif match[:type] == 'name'
         name = match[:match][1]
         next
-      elsif match[:type] == 'tiplink'
-        activity_section = create_activity_section_with_tip(match[:match], tip_match_map)
-      elsif match[:type] == 'remark'
-        activity_section = create_activity_section_with_remark(match[:match], tip_match_map)
       elsif match[:type] == 'pullthrough'
         next if levels.empty?
         pullthrough_match = match[:match]
         # If the syntax takes the form of [code-studio], [code-studio 1-<length>], or [code-studio 2-<length>],
         # add the level activity sections here.
         if pullthrough_match[1].blank? || ([1, 2].include?(pullthrough_match[1]) && levels.length == pullthrough_match[2].to_i)
-          level_sections = create_activity_sections_by_progression(levels, lesson_activity_id, position)
+          level_sections = create_activity_sections_by_progression(levels, lesson_activity_id)
           sections += level_sections
           levels.clear
-          position += level_sections.length
         end
         next
+      elsif match[:type] == 'remark'
```

Comment:
FWIW, moving this down here was a mistake (I removed it then added it back). Sorry for adding more noise to this already hard to read diff 😞